src/main/java/executor/command/CommandSpending.java

```diff
@@ -0,0 +1,108 @@
+package executor.command;
+
+import executor.task.TaskList;
+import interpreter.Parser;
+import ui.Receipt;
+import ui.Ui;
+import ui.Wallet;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class CommandSpending extends Command {
```

Comment:
As above, do refactor the class to CommandAddReceipt/CommandCreateReceipt or something similar so that the Class Name is self-explanatory.